apps/dashboard/app/models/script.rb

```diff
@@ -251,6 +256,11 @@ def cache_file_exists?
     cache_file_path.exist?
   end
 
+  def add_creation_marker(script_dir)
+    creation_marker = Pathname.new(Script.creation_marker_path(script_dir))
+    File.write(creation_marker, { }.to_json) unless creation_marker.exist?
+  end
```

Comment:
I'm confused as to the purpose of this given we have a `created_at` property in the scripts `form.yml`. 

Also - can't we just use `File.ctime(configuration_directory)`?

Reply:
I tried as well with `birthtime`, it is not supported in all system and in my local machine fails. So I decided to add the `created_at`.

Reply:
🤦‍♂️ it's not implemented in Linux either, which I thought for sure it would be.

Well OK - seems like all is well, this marker just needs cleaned up a bit.

Reply:
removed the creation marker and file methods. It is just the `created_at` field now